src/core/init/load_on_media_source.ts

```diff
@@ -176,9 +183,9 @@ export default function createMediaSourceLoader({
             cancelEndOfStream$.next(null);
             return EMPTY;
           case "discontinuity-encountered":
-            if (SourceBuffersStore.isNative(evt.value.bufferType)) {
-              log.warn("Init: Explicit discontinuity seek", evt.value.nextTime);
-              mediaElement.currentTime = evt.value.nextTime;
+            const { bufferType, gap } = evt.value;
+            if (bufferType === undefined || SourceBuffersStore.isNative(bufferType)) {
+              contentDiscontinuities$.next(gap);
```

Comment:
I don't get why a subject is used here...
As, there is no asynchronous task to cancel or whatnot, why not just calling a function here?

Reply:
Yes indeed.